packages/now-cli/src/util/env/add-env-record.ts

```diff
@@ -1,60 +1,34 @@
 import { Output } from '../output';
 import Client from '../client';
 import { Secret, ProjectEnvTarget, ProjectEnvVariableV5 } from '../../types';
-import { customAlphabet } from 'nanoid';
-import slugify from '@sindresorhus/slugify';
 
 export default async function addEnvRecord(
   output: Output,
   client: Client,
   projectId: string,
-  envName: string,
-  envValue: string | undefined,
+  type: 'plain' | 'secret' | 'system',
+  key: string,
+  envValue: string,
   targets: ProjectEnvTarget[]
 ): Promise<void> {
   output.debug(
-    `Adding Environment Variable ${envName} to ${targets.length} targets`
+    `Adding ${type} Environment Variable ${key} to ${targets.length} targets`
   );
 
-  let values: string[] | undefined;
+  let value = envValue;
 
-  if (envValue) {
-    const secrets = await Promise.all(
-      targets.map(target =>
-        client.fetch<Secret>('/v2/now/secrets', {
-          method: 'POST',
-          body: JSON.stringify({
-            name: generateSecretName(envName, target),
-            value: envValue,
-            projectId: projectId,
-            decryptable: target === ProjectEnvTarget.Development,
```

Comment:
Is `decryptable` handled in the secret API now?

Reply:
We're not allowing to create decryptable secrets anymore with advanced env variables. If you want to use a env value in development, it should be a plaintext env variable.